File: BKFoodCourt/app/src/main/java/com/example/bkfoodcourt/ItemAdapter.java
package com.example.bkfoodcourt;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import java.util.List;

public class ItemAdapter extends RecyclerView.Adapter<ItemAdapter.ViewHolder> {

    List<Row> itemList;
    public ItemAdapter(List<Row> itemList) {
        this.itemList = itemList;
    }
    @NonNull
    @Override
    public ItemAdapter.ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
        View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.row, parent, false);
        ViewHolder viewHolder = new ViewHolder(view);

        return viewHolder;
    }

    @Override
    public void onBindViewHolder(@NonNull ItemAdapter.ViewHolder holder, int position) {
        holder.orderID.setText(itemList.get(position).getOrderID());
        holder.userID.setText(itemList.get(position).getUserID());
    }

    @Override
    public int getItemCount() {
        return itemList.size();
    }

    public class ViewHolder extends RecyclerView.ViewHolder {
        TextView orderID, userID;

        public ViewHolder(@NonNull View itemView) {
            super(itemView);
            orderID = itemView.findViewById(R.id.tvOrderID);
            userID = itemView.findViewById(R.id.tvUserID);
        }
    }
}

File: BK_FoodCourt/app/src/main/java/com/example/bk_foodcourt/ItemAdapter.kt
package com.example.bk_foodcourt

import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView


class ItemAdapter(var itemList: List<Row>) :
    RecyclerView.Adapter<ItemAdapter.ViewHolder>() {
    override fun onCreateViewHolder(
        parent: ViewGroup,
        viewType: Int
    ): ViewHolder {
        val view: View =
            LayoutInflater.from(parent.context).inflate(R.layout.row, parent, false)
        return ViewHolder(view)
    }

    override fun onBindViewHolder(
        holder: ViewHolder,
        position: Int
    ) {
        holder.orderID.text = itemList[position].getOrderID()
        holder.userID.text = itemList[position].getUserID()
    }

    override fun getItemCount(): Int {
        return itemList.size
    }

    inner class ViewHolder(itemView: View) :
        RecyclerView.ViewHolder(itemView) {
        var orderID: TextView
        var userID: TextView

        init {
            orderID = itemView.findViewById(R.id.tvOrderID)
            userID = itemView.findViewById(R.id.tvUserID)
        }
    }

}
